orc8r/cloud/go/services/tenants/servicers/protected/servicer_test.go

```diff
@@ -28,6 +28,9 @@ var (
 		Networks: []string{"network_1"},
 	}
 	sampleTenantID      int64 = 3
+	nextTenantID        int64 = 0
+	networkID                 = "networkID"
+	controlProxy              = "controlProxy"
```

Comment:
Where are these used?

Reply:
I guess they were left from the tests before, I removed them.